cloud/rackspace/rax.py

```diff
@@ -285,7 +285,7 @@ def create(module, names=[], flavor=None, image=None, meta={}, key_name=None,
     if user_data:
         config_drive = True
 
-    if user_data and os.path.isfile(user_data):
+    if user_data and os.path.isfile(os.path.expanduser('user_data')):
```

Comment:
The quotes around `user_data` would cause it to expand to `/home/username/user_data` as opposed the actual value of `user_data`.